packages/react-components/src/autocomplete/src/Autocomplete.tsx

```diff
@@ -182,10 +201,12 @@ export function InnerAutocomplete(props) {
         // Usually provided by the field inputs.
         "aria-labelledby": ariaLabelledBy,
         "aria-describedby": ariaDescribedBy,
-        menuProps: { id: menuId, style: { width: menuWidth, ...menuStyle } = {}, ...menuProps } = {},
+        menuProps: { id: menuId, style: { width: menuWidth, ...menuStyle } = { width: undefined }, ...menuProps } = { id: undefined },
```

Comment:
This is the only hack i've done so far. Typescript really struggles in this usecase. 
`Initializer provides no value for this binding element and the binding element has no default value.ts(2525)`
if i remove the width: undefined

Reply:
It's an empty object litteral assignation, while destructuring a property, property which is the result from merging props and omitProps of fieldProps. It's a little more complex than the super common JS example.